app/views/shared/_flash_message.html.erb

```diff
@@ -0,0 +1,13 @@
+<% unless flash.blank? %>
+  <% flash.each do |key, value| %>
+    <% alert_class = case key
```

Comment:
<priority very low> This seems like another place where we could use a translation file and use something like t("alert.css_class.#{key}", default: "alert-"#{key}"). In any case, it doesn't feel to me like it belongs here.

Reply:
twitter-bootstrap-rails gem might take care of this for us.